### ISSUE TITLE
Clean up default phpcs and phpunit configs

### ISSUE THREAD

**Issue by Tjeerd** [ibuildingsnl/flex-recipes#9]
Because we are including symfony test pack, recipes are also run for this pack. This causes some default files to be moved to the project root. These files are duplicated by our own configs.

**Comment by mikelerch** [comment]
Makes sense, not sure how we can achieve this. The only thing you can "run" after an installation is the [post-install-output-configurator](https://github.com/symfony/recipes#post-install-output-configurator), which only prints out the content of .txt files.
